Services/MoneyService.cs

```diff
@@ -35,7 +36,7 @@ public ICurrency DefaultCurrency
             {
                 var defaultIsoCode = _options?.DefaultCurrency;
                 return string.IsNullOrEmpty(defaultIsoCode)
-                    ? Currency.USDollar
+                    ? new Currency(CultureInfo.CurrentCulture)
```

Comment:
See my other comment.

Reply:
I agree with this. Have rolled back this change.